packages/modules/src/modules/banners/common/BannerText.tsx

```diff
@@ -24,6 +24,34 @@ type BannerTextProps = {
     children?: React.ReactNode;
 };
 
+export const createBannerBodyCopy = (
+    paras: (Array<JSX.Element> | JSX.Element)[],
+    highlights: Array<JSX.Element> | JSX.Element | null | undefined,
```

Comment:
this is `highlightedText` everywhere else

Reply:
Renamed variables to reduce confusion